NMakefile

```diff
@@ -0,0 +1,33 @@
+# NMAKE Makefile for bef.
+
+PROGS=bin\bef.exe bin\befprof.exe bin\bef2c.exe
+CC=cl
+O=.obj
+RM_F=del
+
+WARNS= -Wall -wd4820 -wd4131 -wd4459 -wd4242 -wd4244 -wd4996 \
+	-wd4668 -wd4255 -wd4710 -wd4706 -wd4702 -wd4711 -wd4456
```

Comment:
I'd probably recommend adding `-wd5045` to the `WARNS` list to suppress the Spectre mitigation warning that's being reported in the most recent reversions of MSVC.